_dev/src/store/modules/app/state.ts

```diff
@@ -35,6 +35,7 @@ export interface State {
   psxMktgWithGoogleStoreSettingsUrl: string;
   psxMktgWithGoogleOnProductionEnvironment: boolean;
   debugData: DebugData;
+  moduleDebug: ModuleDebug;
```

Comment:
Avoid using debug  word when we actually use the data in the app.

```suggestion
  moduleStatus: ModuleStatus;
```